### ISSUE TITLE
My avatar in web app sidebar is keyboard-focusable but not keyboard-clickable

### ISSUE THREAD

**Issue by hs4man21** [bluesky-social/social-app#641]
**Describe the bug**
I can press the tab key to highlight my avatar with a blue focus ring, indicating it should be clickable with "space" or "enter." Clicking the image with "enter" causes a brief flash but does not take me to my profile. Clicking it with the mouse does work.

We should either add the key event handlers to the avatar for parity with mouse users, or remove the avatar from keyboard focus.

**To Reproduce**

1. Run the web app locally.
2. Use [keyboard controls](https://build.washingtonpost.com/resources/accessibility/keyboard-accessibility#Keyboard%20focus) to navigate to the avatar image and press "enter" key when it is in focus (as shown below with the blue ring) to cause the "flash."
3. Click the avatar with the mouse to confirm that mouse events are in fact handled.

**Expected behavior**

Anywhere there is keyboard focus and a mouse event is handled, [the "enter" key event (for links) or "enter" and "space" (for buttons)](https://build.washingtonpost.com/resources/accessibility/keyboard-accessibility#Limiting%20keyboard%20events%20to%20specific%20keys) should also be handled. In this case, we expect "enter" to take me to my profile just as clicking with mouse does.

**Screenshots**
<img width="135" alt="Screenshot 2023-05-15 at 8 50 51 PM" src="https://github.com/bluesky-social/social-app/assets/38192823/16ff08b2-8139-42e7-ac46-66bbb6a6b316">

https://github.com/bluesky-social/social-app/assets/38192823/04acbb21-0852-400a-a6e4-7702ecd85d3f

**Details**

- Platform: Desktop Chrome MacOS
- Platform version: MacOS: Ventura 13.1, Chrome: 112.0.5615.137
- App version: 1.28(?), not sure where to find web app version number

**Comment by pfrazee** [comment]
Would solving https://github.com/bluesky-social/social-app/issues/645 solve this?

**Comment by hs4man21** [comment]
> Would solving #645 solve this?

Ah sorry, I should have gone back and updated this issue. Yes, they could be solved together depending on the chosen approach. My take is that it would make the most sense to couple `tabIndex="-1"` with `aria-hidden="true"` on that avatar image to save screen reader and keyboard controls users the extra step when there is already a profile button in that left sidebar. If the team agrees with that solution, then I don't see why this issue needs to exist separately from #645.

**Comment by ansh** [comment]
Fixed with #921
